test/verify/check-connection

```diff
@@ -1443,5 +1443,54 @@ server {
         self.allow_journal_messages("couldn't change to runtime dir.*Permission denied")
 
 
+class TestConnectionMultiHost(testlib.MachineCase):
+    provision = {
+        'm1': {"address": "10.111.113.1/20", "memory_mb": 512},
+        'm2': {"address": "10.111.113.2/20", "memory_mb": 512},
+    }
```

Comment:
The code change LGTM, but let's please avoid this super expensive test. This takes 20s to set up for a 0.5s test. Can you please instead add this to test/verify/check-shell-multi-machine `checkDirectLogin`?  You also get the ssh setup and multihost config setting for free.

Reply:
Ok!

Reply:
Done.